ext/php7/integrations/integrations.c

```diff
@@ -26,29 +27,17 @@
     ddtrace_hook_callable(DDTRACE_STRING_LITERAL(class), DDTRACE_STRING_LITERAL(fname), \
                           DDTRACE_STRING_LITERAL(callable), options)
 
-ddtrace_integration ddtrace_integrations[] = {
-    {DDTRACE_INTEGRATION_CAKEPHP, "CAKEPHP", ZEND_STRL("cakephp")},
-    {DDTRACE_INTEGRATION_CODEIGNITER, "CODEIGNITER", ZEND_STRL("codeigniter")},
-    {DDTRACE_INTEGRATION_CURL, "CURL", ZEND_STRL("curl")},
-    {DDTRACE_INTEGRATION_ELASTICSEARCH, "ELASTICSEARCH", ZEND_STRL("elasticsearch")},
-    {DDTRACE_INTEGRATION_ELOQUENT, "ELOQUENT", ZEND_STRL("eloquent")},
-    {DDTRACE_INTEGRATION_GUZZLE, "GUZZLE", ZEND_STRL("guzzle")},
-    {DDTRACE_INTEGRATION_LARAVEL, "LARAVEL", ZEND_STRL("laravel")},
-    {DDTRACE_INTEGRATION_LUMEN, "LUMEN", ZEND_STRL("lumen")},
-    {DDTRACE_INTEGRATION_MEMCACHED, "MEMCACHED", ZEND_STRL("memcached")},
-    {DDTRACE_INTEGRATION_MONGO, "MONGO", ZEND_STRL("mongo")},
-    {DDTRACE_INTEGRATION_MYSQLI, "MYSQLI", ZEND_STRL("mysqli")},
-    {DDTRACE_INTEGRATION_NETTE, "NETTE", ZEND_STRL("nette")},
-    {DDTRACE_INTEGRATION_PDO, "PDO", ZEND_STRL("pdo")},
-    {DDTRACE_INTEGRATION_PHPREDIS, "PHPREDIS", ZEND_STRL("phpredis")},
-    {DDTRACE_INTEGRATION_PREDIS, "PREDIS", ZEND_STRL("predis")},
-    {DDTRACE_INTEGRATION_SLIM, "SLIM", ZEND_STRL("slim")},
-    {DDTRACE_INTEGRATION_SYMFONY, "SYMFONY", ZEND_STRL("symfony")},
-    {DDTRACE_INTEGRATION_WEB, "WEB", ZEND_STRL("web")},
-    {DDTRACE_INTEGRATION_WORDPRESS, "WORDPRESS", ZEND_STRL("wordpress")},
-    {DDTRACE_INTEGRATION_YII, "YII", ZEND_STRL("yii")},
-    {DDTRACE_INTEGRATION_ZENDFRAMEWORK, "ZENDFRAMEWORK", ZEND_STRL("zendframework")},
-};
+#define INTEGRATION(id, int_name)                                      \
```

Comment:
I was confused by what `id` and `int_name` were. Can we use `UCASE` and `lcase` or some other name?

Reply:
id is the identifier for code purposes (e.g. postfixed to the enum names) and int_name is the exposed name.